sequana/canu_scanner.py

```diff
@@ -19,7 +19,7 @@
 import glob
 from sequana import version
 from sequana.lazy import pandas as pd
-from sequana.pylab import pylab
+from sequana.lazy import pylab
```

Comment:
Out of curiosity, what's the difference between pylab and matplotlib.pyplot ?

Reply:
a priori aucune. on peut utiliser l'une ou l'autre.